public/app/features/backup/components/BackupInventory/BackupCreation/BackupCreation.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { useStyles } from '@grafana/ui';
+import moment from 'moment/moment';
+import { BackupCreationProps } from './BackupCreation.types';
+import { getStyles } from './BackupCreation.styles';
+
+export const BackupCreation: FC<BackupCreationProps> = ({ date }) => {
+  const styles = useStyles(getStyles);
+  const momentObj = moment(date);
+  const dayTime = momentObj.format('YYYY[-]MM[-]DD');
+  const hourTime = momentObj.format('HH[:]mm[:]ss');
```

Comment:
I'd consider moving those format strings to a constants file

Reply:
Agree. Done !